crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,234 @@
+function CrowdsourceHinter(runtime, element, data){
+    //switching units back to a previous question will make a second hinter start up.
+    //executeHinter is used to disable the hinter after switching units in an edX course.
+    var executeHinter = true;
+    $(".crowdsourcehinter_block", element).hide();
+
+    if(executeHinter){
+    var isShowingHintFeedback = false;
+    var voted = false;
+    $(".csh_HintsToUse", element).text("");
```

Comment:
Give this a better name. If even you can't tell what this does from the name, you've got a problem. 

Long names are fine. Modern editors have autocomplete.